src/leet2git/leet2git.py

```diff
@@ -0,0 +1,293 @@
+import os
+import shutil
+import time
+from multiprocessing import Process
+from multiprocessing.managers import SyncManager
+from typing import Any, Dict
+
+import click
+from click.exceptions import Abort
+
+from leet2git.config_manager import ConfigManager
+from leet2git.file_handler import FileHandler, generate_files
+from leet2git.leetcode_client import LeetcodeClient
+from leet2git.my_utils import mgr_init, reset_config
+from leet2git.question_db import QuestionData, QuestionDB
+from leet2git.readme_handler import ReadmeHandler
+from leet2git.version import version_info
+
+
+@click.group()
+@click.version_option(version="", message=version_info())
+@click.option(
+    "--source-repository",
+    "-s",
+    default="",
+    help="The path to the folder where the code will be saved. Overrides the default config",
+)
+@click.option(
+    "--language",
+    "-l",
+    default="python3",
+    help="The language to run the command. Overrides the default config",
+)
+@click.pass_context
+def leet2git(ctx, source_repository: str, language: str):
+    cm = ConfigManager()
+    override_config = {}
+    if language:
+        override_config["language"] = language
+    if source_repository:
+        override_config["source_path"] = source_repository
+    cm.load_config(override_config)
+    ctx.obj = cm
+
+
+@leet2git.command()
+@click.argument("id", type=int)
+@click.pass_obj
+def get_question(cm: ConfigManager, id: int):
+    """Generates all the files for a question
+
+    Args:
+        id (int): the question id
+    """
+    qdb: QuestionDB = QuestionDB(cm.config)
+    lc = LeetcodeClient()
+    qdb.load()
+
+    if qdb.check_if_exists(id):
+        click.secho("Question already imported")
+        return
+
+    if not qdb.check_if_slug_is_known(id):
+        qdb.set_id_title_map(lc.get_id_title_map())
+        qdb.save()
+
+    # get question data
+    args: Dict[int, QuestionData] = {}
+    generate_files(args, id, qdb.get_title_from_id(id), lc, time.time(), cm.config)
+
+    if id in args:
+        # store data
+        qdb.add_question(args[id])
+        qdb.save()
+
+        # update readme
+        rh = ReadmeHandler(cm.config)
+        rh.build_readme(qdb.get_sorted_list(sort_by="creation_time"))
+
+
+@leet2git.command()
+@click.argument("id", type=int)
+@click.pass_obj
+def submit_question(cm: ConfigManager, id: int):
+    """Submit a question to Leetcode
+
+    Args:
+        id (int): the question id
+    """
+    qdb: QuestionDB = QuestionDB(cm.config)
+    qdb.load()
+    # create submit file
+    if qdb.check_if_exists(id):
+        file_handler = FileHandler(qdb.get_question(id), cm.config["language"])
```

Comment:
```suggestion
        file_handler = FileHandler(qdb.get_question(id), cm.config)
```
error when using submit-question this should be fixed like this